website/data/docs-nav-data.json

```diff
@@ -1225,6 +1225,15 @@
       }
     ]
   },
+  {
+    "title": "Vault UI",
+    "routes": [
+      {
+        "title": "Custom Messages",
+        "path": "ui/custom-messages"
```

Comment:
Typically, index pages are used for overviews related to the parent directory. Unless we expect to have many docs around custom messages, we should simplify the doc structure here and just have the `ui` folder with a `custom-messages.mdx` file underneath. Unless we have enough content/time to write a brief overview on the Vault UI, we can drop the index page for now.